From 69e9d2cbd41bf6884d2259675cd58b891c6df2de Mon Sep 17 00:00:00 2001
Message-Id: <69e9d2cbd41bf6884d2259675cd58b891c6df2de.1387382496.git.minovotn@redhat.com>
In-Reply-To: <c5386144fbf09f628148101bc674e2421cdd16e3.1387382496.git.minovotn@redhat.com>
References: <c5386144fbf09f628148101bc674e2421cdd16e3.1387382496.git.minovotn@redhat.com>
From: Nigel Croxon <ncroxon@redhat.com>
Date: Thu, 14 Nov 2013 22:53:14 +0100
Subject: [PATCH 38/46] migration: Fix debug print type

RH-Author: Nigel Croxon <ncroxon@redhat.com>
Message-id: <1384469598-13137-39-git-send-email-ncroxon@redhat.com>
Patchwork-id: 55724
O-Subject: [RHEL7.0 PATCH 38/42] migration: Fix debug print type
Bugzilla: 1011720
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>
RH-Acked-by: Amit Shah <amit.shah@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

Bugzilla: 1011720
https://bugzilla.redhat.com/show_bug.cgi?id=1011720

>From commit ID:
commit 5016e2df569bc7d67637060103dd360ed2f0d557
Author: Christoffer Dall <christoffer.dall@linaro.org>
Date:   Fri Aug 23 10:34:16 2013 -0700

    migration: Fix debug print type

    The printf args are uint64_t and with -Werr QEMU doesn't compile with
    migration debugging turned on unless this is fixed.  Fix it.

    Signed-off-by: Christoffer Dall <christoffer.dall@linaro.org>
    Signed-off-by: Juan Quintela <quintela@redhat.com>
---
 migration.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 migration.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/migration.c b/migration.c
index 4f070a3..c0202b4 100644
--- a/migration.c
+++ b/migration.c
@@ -574,7 +574,8 @@ static void *migration_thread(void *opaque)
         if (!qemu_file_rate_limit(s->file)) {
             DPRINTF("iterate\n");
             pending_size = qemu_savevm_state_pending(s->file, max_size);
-            DPRINTF("pending size %lu max %lu\n", pending_size, max_size);
+            DPRINTF("pending size %" PRIu64 " max %" PRIu64 "\n",
+                    pending_size, max_size);
             if (pending_size && pending_size >= max_size) {
                 qemu_savevm_state_iterate(s->file);
             } else {
-- 
1.7.11.7

